results/sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2/SadeemKeywordRetrieval.json

```diff
@@ -0,0 +1,43 @@
+{
+  "dataset_revision": "b76ba10cc7c6a41f9bd69e7dfa57e5ae17ddd761",
+  "mteb_dataset_name": "SadeemKeywordRetrieval",
+  "mteb_version": "1.10.1",
+  "test": {
+    "evaluation_time": 1173.97,
+    "map_at_1": 0.14074,
+    "map_at_10": 0.26039,
+    "map_at_100": 0.26983,
+    "map_at_1000": 0.27068,
+    "map_at_20": 0.26595,
+    "map_at_3": 0.2364,
+    "map_at_5": 0.24938,
+    "mrr_at_1": 0.13308,
+    "mrr_at_10": 0.25653,
+    "mrr_at_100": 0.26595,
+    "mrr_at_1000": 0.2668,
+    "mrr_at_20": 0.26212,
+    "mrr_at_3": 0.23225,
+    "mrr_at_5": 0.24533,
+    "ndcg_at_1": 0.14074,
+    "ndcg_at_10": 0.31558,
+    "ndcg_at_100": 0.36468,
+    "ndcg_at_1000": 0.38949,
+    "ndcg_at_20": 0.33569,
+    "ndcg_at_3": 0.26536,
+    "ndcg_at_5": 0.28884,
+    "precision_at_1": 0.14074,
+    "precision_at_10": 0.04885,
+    "precision_at_100": 0.00726,
+    "precision_at_1000": 0.00093,
+    "precision_at_20": 0.02839,
+    "precision_at_3": 0.11616,
+    "precision_at_5": 0.08114,
+    "recall_at_1": 0.14074,
+    "recall_at_10": 0.48851,
+    "recall_at_100": 0.72595,
+    "recall_at_1000": 0.92604,
+    "recall_at_20": 0.56774,
+    "recall_at_3": 0.34849,
+    "recall_at_5": 0.4057
+  }
+}
```

Comment:
Hi @bakrianoo I saw your testing error 

```
ValueError: The metadata of the following datasets is not filled: ['SadeemKeywordRetrieval']

tests/test_TaskMetadata.py:436: ValueError

```


I faced a similar error. These steps that I did to fix it:

- go to the file `tests/test_TaskMetadata.py`

- add 'SadeemKeywordRetrieval', to the list of _HISTORIC_DATASETS manully.

- save the file and run `make test`